internal/services/compute/shared_image_resource.go

```diff
@@ -187,14 +194,6 @@ func resourceSharedImageCreateUpdate(d *pluginsdk.ResourceData, meta interface{}
 		}
 	}
 
-	var features []compute.GalleryImageFeature
-	if d.Get("trusted_launch_enabled").(bool) {
-		features = append(features, compute.GalleryImageFeature{
-			Name:  utils.String("SecurityType"),
-			Value: utils.String("TrustedLaunch"),
-		})
-	}
```

Comment:
can we remove the expand function here and leave this as it was (adding the new field) - passing values out of `d` in functions makes it more likely we'll introduce bugs when refactoring etc

Reply:
Got it, moved to Create.